lib/OpenQA/Scheduler/Scheduler.pm

```diff
@@ -669,6 +669,44 @@ sub _job_stop_children {
     }
 }
 
+sub _carry_over_labels {
+    my ($job) = @_;
+    if ($job->comments > 0) {
```

Comment:
you are right, this would only hit under special circumstances reg. timing. You could consider this defensive programming. It won't harm and I could easily remove this or try to come up with the cornercase in a test if possible it all ;-) … ok will remove it for now.